### ISSUE TITLE
JS ToDo list: Remove portfolio from introduction

### ISSUE THREAD

**Issue by damon314159** [TheOdinProject/curriculum#27600]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

In the discord we advise (along with the associated command) to not think about portfolio worthy-ing projects, until much later than the organising code section where ToDo lives. However in the introduction to the page it mentions exactly that. 

I feel this is encouraging people to spend too long on areas that aren't the main focus of the task - since they'll be much better poised to make proper portfolio work later on, that will outshine ToDo

### Path

Node / JS

### Lesson Url

https://github.com/TheOdinProject/curriculum/blob/main/javascript/organizing_your_javascript_code/project_todo_list.md

**Comment by CouchofTomato** [comment]
@TheOdinProject/javascript 

Can someone take a look at this please.

**Comment by thatblindgeye** [comment]
I wouldn't be opposed to removing that verbiage and replacing it with something like, "...we’re going to take a minute to slow down and work on another great project that ties these techniques together."

ToDo list I think feels both really early and also not that early to me 😆 There's still a good amount of lessons between it and the Getting Hired section, though, so spending too much time trying to "perfect' a project this early for a portfolio may do a disservice to users.

This would be worth getting input from @TheOdinProject/rails-path as well, since it could be more a curriculum wide thing. Should we be sure to not really mention projects that are "good" for a portfolio/a portfolio in general until it actually becomes relevant in Getting Hired? Like, at that point we tell people, "Okay so now pick your best projects, clean them up, and slap them on your portfolio" (except worded gooder)?

**Comment by MaoShizhong** [comment]
I'm of the opinion that we shouldn't mention about portfolio stuff at least for the Todo list. It's very early in the Node pathway relative to where people often seem to be seriously applying (and being more successful in general apparently). At the very least, I don't think we should be telling people what projects should/can be portfolio pieces. I think people should be working towards that only once they've understood they will soon be seriously applying for jobs and thus have developed a need for a portfolio. I don't think the Todo list, at least where it is in the Node pathway, is a sensible position for many to statt thinking that.

**Comment by damon314159** [comment]
Any traction on this?

**Comment by JoshDevHub** [comment]
Yeah I don't think mentioning the portfolio on this project is necessary from a Rails-path perspective either. I'd be in favor of removing its mention.

**Comment by MaoShizhong** [comment]
Sounds reasonable to action upon then. Assigning you @damon314159
